addons/noactions/stringtable.xml

```diff
@@ -0,0 +1,63 @@
+
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="TAC">
+    <Package name="No Actions">
+        <Key ID="STR_tac_noactions_settings">
+            <English>TAC NoActions</English>
+        </Key>
+        <!-- Eject  -->
```

Comment:
Do strigtable helper tools retain those comments?

Reply:
No idea, will check.

Reply:
Tabler does not remove comments.

Reply:
Great!